src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -46,6 +46,12 @@ interaction type typeIn {
 	method = WebDriverFixture.typeInto(element, locatorStrategy, value)
 }
 
+interaction type isTextOnPage {
+	template = "Is Text" ${value} "available on Web Site" 
```

Comment:
I'd prefer 'Is text ... on page?',  web site would include all pages of the site, right? even those which are currently not shown...

For the GeneralBrowser it would probably even make more sense to name it 'Is text ... on current page?' but I'm not sure whether this justifies another template...

Reply:
Maybe we should add something like 'Is text matching ... on page?' which takes a regular expression as parameter, too.